vllm/model_executor/layers/sampler.py

```diff
@@ -29,6 +27,8 @@ class Sampler(nn.Module):
     parameters (e.g., sampling method, temperature, top-p, top-k, etc.).
     """
 
+    _copy_stream: Optional[torch.cuda.Stream] = None
```

Comment:
Is there any specific reason to make this a global property instead of having a stream per-sampler?